index.js

```diff
@@ -12,3 +13,12 @@ const pagination = document.querySelector('[data-js="pagination"]');
 const maxPage = 1;
 const page = 1;
 const searchQuery = "";
+
+// calling creat card function
+createCharacterCard({
+  image: "https://rickandmortyapi.com/api/character/avatar/1.jpeg",
+  name: "Rick Sanchez",
+  status: "Alive",
+  type: "",
+  episode: [],
+});
```

Comment:
If you follow up on my previous comment, you will return the `li`-element from the `createCharacterCard` function.
You will need to save the return value in a variable and append it to the `cardContainer` afterwards.